packages/fast-usdc/src/type-guards.js

```diff
@@ -93,3 +93,24 @@ export const PoolMetricsShape = {
   totalRepays: AmountShape,
 };
 harden(PoolMetricsShape);
+
+/** @type {TypedPattern<ChainPolicy>} */
+export const ChainPoliciesShape = {
+  nobleContractAddress: EvmHashShape,
+  cctpTokenMessengerAddress: EvmHashShape,
+  confirmations: M.number(),
+  chainName: M.string(),
+  bridgeContractAddress: M.opt(EvmHashShape),
+  eventFilter: M.opt(M.string()),
+  chainType: M.opt(M.number()),
+  isL2: M.opt(M.boolean()),
+};
+harden(ChainPoliciesShape);
+
+/** @type {TypedPattern<FeedPolicy>} */
+export const FeedPolicyShape = {
+  nobleDomainId: M.number(),
```

Comment:
Is there any possibility that this is ever anything besides 4? Should we just constrain it?

```suggestion
  nobleDomainId: 4,
```